lib/passport/google.ts

```diff
@@ -0,0 +1,36 @@
+import { Strategy, Profile } from 'passport-google-oauth20'
```

Comment:
I have a horrible tendency to omit them. Sorry about that.

Reply:
Haha no worries, do you happen to use any plugins like `prettier` or `editorconfig` on your text editor / IDE of your choice? Because, those plugins usually help automatically apply same code style on file saves!

Also, I wasn't 100% sure what @andycmaj prefered as far as `semi: true | false` on this specific proj.